src/app/components/Forms/PopupForm.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import styled from 'styled-components'
+import Button from '../Button/Button'
+import OverlayWrapper from '../OverlayWrapper/OverlayWrapper'
+
+type FormProps = {
+  onCancel: () => void
+  onSubmit: (event: React.FormEvent<HTMLFormElement>) => void
+  SubmitText: string
+  children: React.ReactNode
+}
+
+export default function Form({
+  onCancel,
+  onSubmit,
+  children,
+  SubmitText,
+}: FormProps): JSX.Element {
+  return (
+    <OverlayWrapper onReturn={onCancel}>
+      <FormContainer onSubmit={onSubmit}>
+        <FormTitle>{SubmitText}</FormTitle>
+        {children}
+        <ButtonContainer>
+          <Button type="button" onClick={onCancel}>
+            Cancel
+          </Button>
+          <Button highlight>{SubmitText}</Button>
```

Comment:
What is the use of a button which does nothing on click?

Reply:
its a submit button, the function is defined on the Form element.